CHANGELOG.md

```diff
@@ -101,6 +117,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Updates migration script with new `migrateAndOverwrite` and
     `migrateOnlyFiles` options.
 
+### Breaking Changes
+
+- **CUMULUS-2903**
```

Comment:
This isn't legit, but it's mis-merged in main.  The fix needs to be applied there and brought forward to feature.

Reply:
Yup. I don't want to talk about it 😆 

https://github.com/nasa/cumulus/pull/2961

Reply:
https://github.com/nasa/cumulus/pull/2961 is merged. If you want to pull the latest from master into this branch that'll take care of it here. Otherwise I can merge just that changelog branch into rds-phase-3 after this PR.